scalabel/vis/label.py

```diff
@@ -419,9 +419,9 @@ def _draw_ctrl_points(
                         )
                     )
 
-    def _get_node_color(
+    def _get_node_color(  # pylint: disable=no-self-use
```

Comment:
Better to make it a standalone function instead of a class method

Reply:
Fixed